tcms/testruns/templates/testruns/get.html

```diff
@@ -262,6 +264,47 @@ <h5><span class="test-executions-count"></span> Results</h5>
     </div> <!-- /row -->
 </div>
 
+<div class="modal fade" id="add-link-modal" tabindex="-1" role="dialog" aria-labelledby="add-hyperlink-modal-title" aria-hidden="true">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-hidden="true" aria-label="Close">
+          <span class="pficon pficon-close"></span>
+        </button>
+        <h4 class="modal-title" id="add-hyperlink-modal-title">Add hyperlink</h4>
+      </div>
+      <form class="form-horizontal add-hyperlink-form">
+        <div class="modal-body">
+          <div class="form-group">
+            <label class="col-sm-3 control-label" for="{{ link_form.url.id_for_label }}">URL</label>
+            <div class="col-sm-9">
+                <input type="url" name="{{ link_form.url.name }}" maxlength="{{ link_form.url.field.max_length }}" {% if link_form.url.field.required %}required{% endif %} id="{{ link_form.url.id_for_label }}" class="form-control">
+            </div>
+          </div>
+          <div class="form-group">
+            <label class="col-sm-3 control-label" for="{{ link_form.name.id_for_label }}">Name</label>
+            <div class="col-sm-9">
+                <input type="text" name="{{ link_form.name.name }}" maxlength="{{ link_form.name.field.max_length }}" {% if link_form.name.field.required %}required{% endif %} id="{{ link_form.name.id_for_label }}" class="form-control">
+            </div>
+          </div>
+          <div class="form-group">
+            <label class="col-sm-3 control-label" for="defectCheckbox">Is a defect</label>
+            <input class="col-sm-9 bootstrap-switch" type="checkbox" id="defectCheckbox"/>
+          </div>
```

Comment:
I can notice that only the `-sm-` classes are used while other pages use -lg- and -md- classes and then -sm- if needed. Not quite sure if that has anything to do with the visual discrepancies.

All labels should be vertically aligned on the right, labels for the on/off buttons are not.

All controls should be vertically aligned on their left, on/off buttons aren't aligned with the text fields above.

Also when resizing on smaller screens these buttons don't look very good b/c they align with the text on 1 row and the texts don't align vertically even if there is space.

Reply:
See how the col- classes are placed above, not on the input element itself but on a surrounding div.

Reply:
Still unresolved.